.travis.yml

```diff
@@ -109,14 +110,13 @@ script:
 
 after_success:
   - find . -name '*.gcno' -print
-  - find . -name '*.gcda' -print
   - gcov-5 --version
   - bash <(curl -s https://codecov.io/bash) -x $(which gcov-5)
 
 notifications:
   webhooks:
     urls:
-      - $GITTERHOOK_URL
+      - https://webhooks.gitter.im/e/93dbafbdf76c1732a623
```

Comment:
Ideally this would be a travis-ci encrypted variable to prevent someone from flooding the notifications. However, it seems Travis secure environment variables won't work for this, and I can't figure out how to encrypt it correctly with the travis gem. (`gem install travis` or if needed `sudo gem install travis` to install this useful ruby gem with helper functions including linting your .travis.yml)